pkg/plugin/client/client.go

```diff
@@ -66,8 +74,13 @@ func (*clientFactory) New(aspectplugin loader.AspectPlugin, streams ioutils.Stre
 	}
 
 	res := &PluginInstance{
-		Plugin:   rawplugin.(plugin.Plugin),
-		Provider: goclient,
+		Plugin:         rawplugin.(plugin.Plugin),
+		Provider:       goclient,
+		CustomCommands: map[string]*plugin.Command{},
```

Comment:
Where does this one get used? (I'm probably just not finding it..?)

Reply:
Its not. Good catch. I think I had added this about midway through my rebase with your changes (when I would have still needed it) and then never removed it when I got to my later changes where its was no longer needed